spec/models/search/pseud_query_spec.rb

```diff
@@ -22,61 +22,55 @@
       pseud_bar_2: create(:pseud, user: users[:user_bar], name: "foo"),
       pseud_aisha: create(:pseud, user: users[:user_aisha], name: "عيشة")
     }
-    update_and_refresh_indexes("pseud", 1)
```

Comment:
We set the number of shards to 1 to get stable order of results for tag search (#3224). The new hooks don't do this (default multiple shards), so the order varies. Can we bring this back?

Reply:
But all of our indexes have multiple shards on production. Shouldn't we want the tests to mimic the deployed behavior?

Reply:
Generally yes, though in this case, we want to test that pseud searches favor exact matches more:

https://github.com/otwcode/otwarchive/blob/67b7b149d41ef626aca3596f2b1448b34d0cdc05/app/models/search/tag_query.rb#L122

It's hard to test TagQuery's behavior on multiple indices, so we check that its behavior on one index hasn't changed and assume that its behavior on multiple indices hasn't changed, unit-test-like.

Reply:
The problem wasn't the number of shards per se but how ES distributes and then gathers up data for sorting in the presence of tiny amounts of data (official explanations here: https://www.elastic.co/guide/en/elasticsearch/reference/current/consistent-scoring.html and here: https://www.elastic.co/guide/en/elasticsearch/guide/2.x/relevance-is-broken.html). The original discussion leading to the change I made in #3224 is here in one of our private channels: https://transformativeworks.slack.com/archives/G2CRYKP8W/p1520496182000310.

If we want the tests to reflect production and use multiple shards, we would need to load a lot more data in order to get a consistent relevance score that could be used for an ordering test, so our options were to load more data or use one shard. Or I guess, stop testing ordering 😬